src/ErrorElement.tsx

```diff
@@ -0,0 +1,72 @@
+import {
+  Button,
+  Flex,
+  Heading,
+  Link as RadixLink,
+  Text,
+} from '@radix-ui/themes'
+import { css } from '@emotion/react'
+
+import GrotCrashed from '@/assets/grot-crashed.svg'
+import { ArrowLeftIcon } from '@radix-ui/react-icons'
+
+const handleCreateIssue = () => {
+  window.studio.browser.openExternalLink(
+    'https://github.com/grafana/k6-studio/issues'
+  )
+}
+
+const handleOpenLogs = () => {
+  window.studio.app.openApplicationLog()
+}
+
+const handleRestart = () => {
+  window.location.replace('/')
+}
+
+export function ErrorElement() {
+  return (
+    <Flex height="100dvh" justify="center" align="center" p="8">
+      <Flex direction="column" align="center" gap="4">
+        <Heading
+          size="8"
+          css={css`
+            font-weight: 400;
+          `}
+        >
+          Unexpected error
+        </Heading>
+        <img
+          src={GrotCrashed}
+          role="presentation"
+          css={css`
+            width: 375px;
+          `}
+        />
+        <Text
+          size="2"
+          color="gray"
+          align="center"
+          css={css`
+            max-width: 560px;
+          `}
+        >
+          We apologize for the inconvenience. Please help us improve our service
```

Comment:
should it be `service` -> `application` ? 🤔

Reply:
Yes, sounds better!